charts/catalog/templates/rbac.yaml

```diff
@@ -89,10 +89,10 @@ items:
     resources: ["serviceclasses"]
     verbs:     ["get","list","watch","create","patch","update","delete"]
   - apiGroups: ["servicecatalog.k8s.io"]
-    resources: ["bindings","brokers","instances"]
+    resources: ["servicebindings","servicebrokers","serviceinstances"]
```

Comment:
Needs to be `serviceinstancecredentials` instead of `servicebindings`.  Looks good otherwise.